nebula-logger/core/main/log-management/lwc/logEntryEventStream/logEntryEventStream.css

```diff
@@ -17,8 +15,9 @@
     font-family: 'Lucida Console', 'Courier New', monospace;
     text-indent: -32px;
     padding-left: 32px;
-    height: calc(100vh - 300px);
-    margin-right: 8px;
+    /*height: calc(100vh - 300px);
+    margin-right: 8px;*/
```

Comment:
Do you think we can delete these lines (instead of leaving them commented-out)? Visually, your changes look amazing, so I'm not sure we need this CSS anymore.